vite.config.js

```diff
@@ -0,0 +1,17 @@
+import { join } from 'path'
+import { fileURLToPath } from 'url'
+import { defineConfig } from 'vite'
+import vitePugPlugin from 'vite-plugin-pug-transformer'
+
+const ROOT = join(fileURLToPath(import.meta.url), '..')
```

Comment:
Maybe move path constants to separate file and import they where needed?

Reply:
For now they declared in 3 or more files. So if file structure changes more edits will be required